### DISCUSSION TITLE
vite中打包无法访问

### DISCUSSION THREAD

**Discussion by QL9527** [zouyaoji/vue-cesium#227] [Q&A]
![image](https://user-images.githubusercontent.com/38763105/178901636-e95fa8d9-78b6-4099-938d-02ff92fdb99a.png)
vite中打包后，有人遇到过吗，难道是cesiumJs引入失败。本地没问题

**Comment by zouyaoji** [comment]
看着应该是 cesiumPath 打包后错了哈。
